ntbs-ui-tests/Features/CreateNotifications.feature

```diff
@@ -4,24 +4,27 @@ Feature: Notification creation
     Background: Create new notification
         Given I am on the Search page
         When I enter 1 into 'SearchParameters_IdFilter'
-        And I click on 'search-button'
+        And I click on the 'search-button' button
         Then I should be on the Search page
-        When I click on 'create-button'
+        When I click on the 'create-button' button
         Then I should be on the PatientDetails page
 
     Scenario: Create and submit notification without errors
-        # Patient page
-        When I enter Test into 'Patient_GivenName'
-        And I enter User into 'Patient_FamilyName'
+        # PatientDetails page
+        When I enter Test into 'PatientDetails_GivenName'
+        And I enter User into 'PatientDetails_FamilyName'
         And I enter 1 into 'FormattedDob_Day'
         And I enter 1 into 'FormattedDob_Month'
         And I enter 1970 into 'FormattedDob_Year'
         And I select radio value 'sexId-1'
-        And I select 1 for 'Patient_EthnicityId'
+        And I select 1 for 'PatientDetails_EthnicityId'
         And I select radio value 'nhs-number-unknown'
-        And I select 1 for 'Patient_CountryId'
+        And I enter Afghanistan into 'PatientDetails_CountryId'
+        # Need to click away from autocomplete select dropdown for it to populate actual select
+        And I click away from element 'PatientDetails_CountryId'

```

Comment:
Seems like could make that part of the `enter` bit - bundle it all as a single step? Perhaps we can differentiate it from normal inputs by
`I enter Afghanistan into 'PatientDetails_CountryId' autocomplete`? Then it could include the `wait` too, and be.